README.markdown

```diff
@@ -652,6 +653,33 @@ private func makeLocationManager() -> CLLocationManager {
   - `[unowned self]` is not required here. A retain cycle is not created.
   - Location manager has a side-effect for popping up UI to ask the user for permission so fine grain control makes sense here.
 
+### OptionSetTypes
+
+When using OptionSetTypes, signify the "no" option using the empty array option rather than the rawValue initializer. For example, given:
+
+```swift
+struct PackagingOptions : OptionSetType {
+    let rawValue: Int
+    init(rawValue: Int) { self.rawValue = rawValue }
+    static let Box = PackagingOptions(rawValue: 1)
+    static let Carton = PackagingOptions(rawValue: 2)
+}
+
+let options = PackagingOptions.Box
```

Comment:
Not sure this needs to be here. In any sense, I wonder where this line even lands in our style guide with regards to type inference.

I imagine something like this:
```swift
let options: PackagingOptions = [.box, .carton]
```

I'll make an issue.